.github/workflows/scheduled_build.yml

```diff
@@ -39,6 +39,6 @@ jobs:
       - name: Push to "Roave/SecurityAdvisories"
         run: |
           cd build/roave-security-advisories
-          git push "https://${PUSH_GITHUB_TOKEN}:x-oauth-basic@github.com/Roave/SecurityAdvisories.git" master
+          git push "https://${PUSH_GITHUB_TOKEN}:x-oauth-basic@github.com/Roave/SecurityAdvisories.git" latest master
```

Comment:
We can push `latest` and `latest:master`, so that we publish both `master` (for BC) and `latest` (new name)

Reply:
I did not know about `<src>:<dst>` notation so I learned something new!  : )

Reply:
Good! \o/